src/settings_type.h

```diff
@@ -21,6 +21,15 @@
 #include "zoom_type.h"
 #include "openttd.h"
 
+/* Used to validate sizes of "max" value in settings. */
+const size_t size_SLE_UINT8 = UINT8_MAX;
```

Comment:
variable names aren't particularly clear...

Reply:
suggestions?

Reply:
As in, size_SLE_UINT8 to me suggests the size of the variable itself (1), rather than the maximum value

Reply:
Again, suggestions? :D Help me out here ...

Reply:
max_SLE_UINT8 is the obvious sugestion...

Reply:
What is obvious for you doesn't mean is obvious to others ;)

As discussed, `MAX_` it is!